src/main/resources/elastic-rest.yml

```diff
@@ -139,7 +139,7 @@ alias:
 
 allocation:
   versions:
-    ">= 0.9.0": "/_cat/allocation?v&format=json"
+    ">= 0.9.0": "/_cat/allocation?v&pretty&format=json"
```

Comment:
Have you validated that these work with older versions?

Reply:
I did not 😞 - AFAICT `pretty` is used / ignored but doesn't cause calls to fail. Is that incorrect?